source/codegen/metadata/nisync/attributes.py

```diff
@@ -1,3 +1,333 @@
+    },
+    1150503: {
+        'name': 'CLKOUT_GAIN_ENABLE'
+    },
+    1150504: {
+        'name': 'PXICLK10_PRESENT'
+    },
+    1150505: {
+        'name': 'CLKIN_ATTENUATION_DISABLE'
+    },
+    1150600: {
+        'name': 'USER_LED_STATE'
+    },
+    1150700: {
+        'name': '1588_IP_ADDRESS'
+    },
+    1150712: {
+        'name': '1588_CLOCK_STATE'
+    },
+    1150729: {
+        'name': '1588_CLOCK_ID'
+    },
+    1150730: {
+        'name': '1588_CLOCK_CLASS'
+    },
+    1150731: {
+        'name': '1588_CLOCK_ACCURACY'
+    },
+    1150732: {
+        'name': '1588_PRIORITY1'
+    },
+    1150733: {
+        'name': '1588_PRIORITY2'
+    },
+    1150734: {
+        'name': '1588_GRANDMASTER_CLOCK_ID'
+    },
+    1150735: {
+        'name': '1588_GRANDMASTER_CLOCK_CLASS'
+    },
+    1150736: {
+        'name': '1588_GRANDMASTER_CLOCK_ACCURACY'
+    },
+    1150737: {
+        'name': '1588_GRANDMASTER_PRIORITY1'
+    },
+    1150738: {
+        'name': '1588_GRANDMASTER_PRIORITY2'
+    },
+    1150716: {
+        'name': '1588_STEPS_TO_GRANDMASTER'
+    },
+    1150739: {
+        'name': '1588_LOG_SYNC_INTERVAL'
+    },
+    1150740: {
+        'name': '1588_MEAN_PATH_DELAY'
+    },
+    1150741: {
+        'name': '1588_GRANDMASTER_IP_ADDRESS'
+    },
+    1150742: {
+        'name': '1588_BMCA_MODE'
+    },
+    1150743: {
+        'name': '1588_INTERFACE_NAME'
+    },
+    1150718: {
+        'name': '1588_TIMESTAMP_BUF_SIZE'
+    },
+    1150719: {
+        'name': '1588_AVAIL_TIMESTAMPS'
+    },
+    1150720: {
+        'name': '1588_CLK_RESOLUTION'
+    },
+    1150800: {
+        'name': 'TIMEREF_PRESENT'
+    },
+    1150801: {
+        'name': 'TIMEREF_CURRENT'
+    },
+    1150802: {
+        'name': 'TIMEREF_OFFSET'
+    },
+    1150808: {
+        'name': 'TIMEREF_OFFSET_NS'
+    },
+    1150804: {
+        'name': 'TIMEREF_CORRECTION'
+    },
+    1150805: {
+        'name': 'TIMEREF_UTC_OFFSET'
+    },
+    1150806: {
+        'name': 'TIMEREF_UTC_OFFSET_VALID'
+    },
+    1150807: {
+        'name': 'TIMEREF_LAST_SYNC_ID'
+    },
+    1150809: {
+        'name': 'TIMEREF_SELECTED_TYPE'
+    },
+    1150810: {
+        'name': 'TIMEREF_TYPE'
+    },
+    1150811: {
+        'name': 'TIMEREF_SELECTED_NAME'
+    },
+    1150812: {
+        'name': 'TIMEREF_ENABLED'
+    },
+    1150813: {
+        'name': 'TIMEREF_IS_SELECTED'
+    },
+    1150900: {
+        'name': 'GPS_ANTENNA_CONNECTED'
+    },
+    1150901: {
+        'name': 'GPS_RECALCULATE_POSITION'
+    },
+    1150902: {
+        'name': 'GPS_SATELLITES_AVAILABLE'
+    },
+    1150903: {
+        'name': 'GPS_SELF_SURVEY'
+    },
+    1150904: {
+        'name': 'GPS_MOBILE_MODE'
+    },
+    1150905: {
+        'name': 'GPS_STATUS'
+    },
+    1150803: {
+        'name': 'TIMEREF_CLK_ADJ_OFFSET'
+    },
+    1150906: {
+        'name': 'GPS_UTC_OFFSET'
+    },
+    1151000: {
+        'name': 'IRIG_TAI_OFFSET'
+    },
```

Comment:
Since these are deprecated, let's not include them. See nisync.h,
```c++
/* Deprecated Attributes - Do not use */
#define NISYNC_ATTR_TIMEREF_CLK_ADJ_OFFSET   (NISYNC_ATTR_BASE + 803L)
#define NISYNC_ATTR_GPS_UTC_OFFSET           (NISYNC_ATTR_BASE + 906L)
#define NISYNC_ATTR_IRIG_TAI_OFFSET          (NISYNC_ATTR_BASE + 1000L)
```

Reply:
Removed deprecated attributes